### ISSUE TITLE
can't silence notifications

### ISSUE THREAD

**Issue by brainchild0** [k9mail/k-9#4482]
# Expected behavior
Notifications can be configured either to be silent or to include a sound.

# Actual behavior
Sound is played whenever notifications are enabled and a new message is found.

# Environment
K-9 Mail version: 5.7.4
Android version: 9
Account type: IMAP

# Notes

Possibly related to  #4081.

**Comment by cketti** [comment]
Use 'Account settings > Notifications > Notification settings' to configure how to be notified.

The UI is specific to your Android version. But you'll most likely see the notification channels "Messages" and "Miscellaneous". On Vanilla Android you can click on "Messages" (the text, not the checkbox) to configure how to be notified.

**Comment by brainchild0** [comment]
Based on the below screen capture (account information redacted), which combinations of settings would you suggest I try to enable *visual notifications only*? 

---

![Screenshot_20200127-114741_Settings](https://user-images.githubusercontent.com/34869175/73195626-31395a80-40fc-11ea-904a-ec00eb2b0f6e.png)

**Comment by cketti** [comment]
As I wrote before:
>  On Vanilla Android you can click on "Messages" (the text, not the checkbox) to configure how to be notified.

In the future please use the [mailing list](http://groups.google.com/group/k-9-mail) for support requests.

**Comment by brainchild0** [comment]
I now understand. The notification details can be finely tuned through  the submenu that appears from tapping the _word_ not the _checkbox_.

While it is good that I am now familiar with this feature of the Android platform, the plain fact is that now is the first time I have had to use it. All other messaging applications I have used expose this configurability directly through the application menus. I suspect I am not the first user unfamiliar with this use of the Android system, and it may be worth considering an enhancement that exposes these options directly in the application menus.

Because this topic is a request for modification to the application, whether or not the enhancement is ultimately accepted,  I would suggest that the issue tracker not the mailing list is the appropriate location for this discussion.

Thank you for your comments so far.
